src/blocks/infoBlock.vue

```diff
@@ -109,63 +121,60 @@ import Vue from "vue";
 import utils from "@/plugins/utils";
 import { GweiBalance, TokenPrices, TotalByToken, TransactionData, TransactionFee } from "@/types/index";
 import { BigNumber } from "ethers";
-import { ETHER_NETWORK_NAME, ETHER_PRODUCTION, ZK_IS_BETA } from "~/plugins/build";
+import Logo from "@/blocks/logo.vue";
+import { ETHER_NETWORK_NAME, ETHER_PRODUCTION } from "~/plugins/build";
 
 export default Vue.extend({
+  components: {
+    Logo,
+  },
   data() {
     return {
       totalOpened: false,
       feesOpened: false,
-      links: [
-        {
-          title: "Terms",
-          url: "https://zksync.io/legal/terms.html#introduction",
-        },
-        {
-          title: "Privacy",
-          url: "https://zksync.io/legal/privacy.html#introduction",
-        },
-        {
-          title: "Docs",
-          url: "https://zksync.io/api/sdk/checkout/tutorial.html#getting-started",
-        },
-        {
-          title: "zkLink",
-          url: "https://link.zksync.io/",
-        },
-        {
-          title: "zkSync",
-          url: "https://zksync.io",
-        },
-      ],
     };
   },
   computed: {
+    isInfoAvailable(): boolean {
+      if (this.transactionData.transactions.length < 1)
+      {
+        return false;
+      }
+      return this.$store.getters["checkout/getErrorState"] !== true
+    },
+    zkMint(): string {
+      return `https://mint${this.network==='ropsten' ? '-ropsten' : ''}.zksync.dev`;
+    },
     loggedIn(): boolean {
       return this.$store.getters["account/loggedIn"];
     },
     network(): string {
       return ETHER_NETWORK_NAME;
     },
-    isBeta(): boolean {
-      return ZK_IS_BETA;
-    },
     isMainnet(): boolean {
       return ETHER_PRODUCTION;
     },
     transactionData(): TransactionData {
       return this.$store.getters["checkout/getTransactionData"];
     },
     allFees(): Array<TransactionFee> {
+      if (!this.loggedIn) {
+        return this.$store.getters["checkout/getAllFees"].filter((item: TransactionFee) => item.key !== "changePubKey");
+      }
       return this.$store.getters["checkout/getAllFees"];
     },
     totalFees(): GweiBalance {
       const allFees = this.allFees;
       let totalFeeBigNum = BigNumber.from("0");
+
       for (const item of allFees) {
+        if (item.key === "changePubKey" && !this.loggedIn) {
```

Comment:
Because the user is not yet logged in, why would we show account activation. Theoretically, the account activation fee could still be there if the user logins out, that's why we need a check.